composer.json

```
@@ -22,7 +22,8 @@
     },
   "require": {
     "composer/installers": "^2.3",
-    "mautic/core-lib": "^7.0"
+    "mautic/core-lib": "^7.0",
+    "twig/string-extra": "^3.21"
```

```suggestion
    "mautic/core-lib": "^7.0"
```

This change needs to be reverted. There's no need to add it here since it's already present in `app/composer.json`.